terraform/node_module_expand.go

```diff
@@ -41,7 +38,7 @@ func (n *nodeExpandModule) ModulePath() addrs.Module {
 	// This node represents the module call within a module,
```

Comment:
Is this comment still accurate given the change? I have a hard time wrapping my head around this section so I wouldn't be surprised if it is.

Reply:
Thanks, good catch!